### PR TITLE
feat(readme): copy from crate

### LINKED ISSUE #30: copy over readme from crate (~for now~)
for 0.1.0 let's just copy over the README.md from the crate

there's great convo in #7 and i'd like to move to something more like that for 0.2.0 or greater.

### PR DESCRIPTION
**Pull request by ashleygwilliams**
fixes #30 and #24

### CODE REVIEW

File: src/readme.rs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+use std::fs;
+use std::fs::File;
+use failure::Error;
+
+pub fn copy_from_crate(path: &str) -> Result<(), Error> {
+    let path_to_crate_readme = format!("{}/README.md", path);
```
Comment (sendilkumarn) [inline src/readme.rs]: nit: can rename to `crate_readme_path`

Comment (ashleygwilliams) [reply src/readme.rs]: lol yes yes this is a good idea

Comment (ashleygwilliams) [reply src/readme.rs]: i'm a long variable names monster 🐉

**Comment by ashleygwilliams** [comment]
whew this is a big one but also it makes things a LOT nicer.

@yoshuawuyts this backs out the dir_diff stuff- i think i'd like to have a larger discussion on your vision for how that works? i'm not entirely sure it was testing what we wanted, so for now i'm doing this really naive `fs::metadata` testing instead. wanna start an issue and maybe spell it out a bit more? (i'm very open to whatever strategy but it wasn't working for my brain at the moment and i really want to get this merged)

**Pull request merged**